pkg/compilers/includeos/includeos_qemu.go

```diff
@@ -50,3 +50,7 @@ func (i *IncludeosQemuCompiler) findFirstImageFile(directory string) (string, er
 	}
 	return "", errors.New("no image file found", goerrors.New("end of dir"))
 }
+
+func (r *IncludeosQemuCompiler) Usage() string {
+	return "TODO: usage for IncludeosQemuCompiler"
```

Comment:
Write something like "No usage information available" or something like that. The TODO is for you or @ilackarms, not for the user interested in understanding how to use the compiler.